projects/demo-integrations/src/tests/kit/number/number-prefix-postfix.cy.ts

```diff
@@ -1,14 +1,18 @@
-import {DemoPath} from '@demo/constants';
+import {
+    CHAR_EM_DASH,
+    CHAR_EN_DASH,
+    CHAR_HYPHEN,
+    CHAR_MINUS,
+} from 'projects/kit/src/lib/constants';
```

Comment:
`projects/` I don't remember, is it okey path for tests?

Reply:
We have some problems with playwright (Taiga UI project) but it is okay for Cypress (it works locally and in CI)